### PR TITLE
use ansible handlers (#25)

### PR DESCRIPTION
**Pull request by pmauduit**
This to avoid restarting nginx when not necessary

### CODE REVIEW

File: qgis4/ansible/roles/certification/tasks/main.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: Import Nginx role
+  include_role:
```
Comment (elemoine) [inline qgis4/ansible/roles/certification/tasks/main.yml]: I am not fully sure but using `include_role` in each depending role won't the nginx role be executed multiple times when running the full playbook? I wonder if relying on [role dependencies](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-dependencies) wouldn't be a better option for that matter.

Comment (pmauduit) [reply qgis4/ansible/roles/certification/tasks/main.yml]: It seems so indeed. Using dependencies instead leads to have nginx apt step executed only once. Checking 2 or 3 more things, and I'll update.

File: qgis4/ansible/roles/certification/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - role: nginx
```
Comment (elemoine) [inline qgis4/ansible/roles/certification/meta/main.yml]: I am super picky here but do you mind removing the two whitespaces before the `-`. I also hate unnecessary spaces/indents :-)

**Comment by elemoine** [comment]
This looks great. I am just nitpicking on final, super minor, thing :-)

File: qgis4/ansible/roles/certification/meta/main.yml (hunk 2 above)
Comment (pmauduit) [reply qgis4/ansible/roles/certification/meta/main.yml]: sure ... ;-)

**Comment by elemoine** [comment]
@pblottiere can you please merge this?

**Pull request merged**